portfolio/src/main/java/com/google/sps/servlets/AuthServlet.java

```diff
@@ -0,0 +1,49 @@
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+
+
```

Comment:
nit: One blank line should be sufficient here

See https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace